src/components/ChatLog.jsx

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ( props) => {
+  const {entries, onclickChat} = props
+  const chatComponents = props.entries.map((chat, i) => {
+    return (
+      <ChatEntry
+        id ={chat.id}
+        sender={chat.sender}
+        body={chat.body}
+        timeStamp={chat.timeStamp}
+        liked={chat.liked}
+        onclickChat={onclickChat}
+        key ={i}
+      />
+    );
+  });
+  return (
+    <div>{chatComponents}</div>
```

Comment:
Nice job capturing the return value from invoking `map` in the variable `chatComponents` and using that variable here. This is typical of what you'd see in industry!